database/database.go

```diff
@@ -4,41 +4,37 @@ import (
 	"context"
 	"time"
 
-	"github.com/dezh-tech/immortal/types"
 	"go.mongodb.org/mongo-driver/bson"
 	"go.mongodb.org/mongo-driver/mongo"
 	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
-var KindToCollectionName = map[types.Kind]string{
-	types.KindTextNote:        "text_notes",
-	types.KindReaction:        "reactions",
-	types.KindProfileMetadata: "profile_metadatas",
-}
-
 type Database struct {
 	DBName       string
 	QueryTimeout time.Duration
 	Client       *mongo.Client
 }
 
-func New(cfg Config) (*Database, error) {
+func Connect(cfg Config) (*Database, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Duration(cfg.ConnectionTimeout)*time.Millisecond)
```

Comment:
Add a new timeout to the configuration to initialize the connection. "connectionTimeout" is used to manage connection pool timeout, not to initialize the connection.